invent-code-1/main.cpp

```diff
@@ -4,4 +4,5 @@ using namespace std;
 
 int main() {
   // Write code here.
+  cout << "Texas A&M is better than UT. " << "Taxation is theft. " << "Epstein didn't kill himself." << endl;
```

Comment:
can you make this multiple lines? this is a bit long.

Reply:
Fixed.